gui/Tooltip.cpp

```diff
@@ -71,22 +75,25 @@ void Tooltip::drawDialog(DrawLayer layerToDraw) {
 
 	Dialog::drawDialog(layerToDraw);
 
-	int16 textX = _x + 3; // including 2px padding and 1px original code shift
+	int16 textX = g_gui.useRTL() ? _x - 3 : _x + 3; // including 2px padding and 1px original code shift
 	int16 textY = _y + 3;
+
+	Graphics::TextAlign textAlignment = g_gui.useRTL() ? Graphics::kTextAlignRight : Graphics::kTextAlignLeft;
+
 	for (Common::StringArray::const_iterator i = _wrappedLines.begin(); i != _wrappedLines.end(); ++i, ++num) {
 		g_gui.theme()->drawText(
 			Common::Rect(textX, textY + num * h, textX + _w, textY + (num + 1) * h),
 			*i,
 			ThemeEngine::kStateEnabled,
-			Graphics::kTextAlignLeft,
+			textAlignment,
 			ThemeEngine::kTextInversionNone,
 			0,
 			false,
 			ThemeEngine::kFontStyleTooltip,
 			ThemeEngine::kFontColorNormal,
 			false
 		);
+		}
```

Comment:
This is a no-op whitespace change, please revert, same goes with the change below.